.github/workflows/e2e-tests.yml

```diff
@@ -9,16 +9,21 @@ on:
   workflow_dispatch:
 jobs:
   e2e-tests:
+    strategy:
```

Comment:
Another thing I realize that might be a show stopper is the fact we'd have to remove the e2e-tests job as a required check to allow for the individual matrix strategy sub-jobs to run, increasing reviewer burden. Any ideas on how we'd solve this? It doesn't look like github has support for aggregating matrix results into a single result, so a composite action may/may not be usable. Maybe utilizing an action that labels PRs with a hold if any of the matrix runs fail? Thoughts?

Reply:
cc @joelanford @perdasilva in case any of you have thoughts on how to solve this too.

Reply:
https://github.community/t/get-status-of-parallel-jobs-into-a-single-webhook-payload/18077/2 looks fairly promising so maybe another job can be a workaround.

Reply:
I ended up just tacking on another job to this existing workflow, and renaming the jobs defined in this workflow, such that the aggregate status job is named `e2e-tests` so we don't have to change any branch protection rules that are already defined for this repository.